subprojects/robotpy-wpimath/wpimath/units.py

```diff
@@ -2,7 +2,10 @@
 # Open Source Software; you can modify and/or share it under the terms of
 # the WPILib BSD license file in the root directory of this project.
 
+from __future__ import annotations
```

Comment:
IIUC `from __future__ import annotations` is deprecated by [PEP 649](https://peps.python.org/pep-0649/)

Reply:
I don't care.